src/test/unittest/utils.py

```diff
@@ -46,3 +47,73 @@
 GiB = 2 ** 30
 TiB = 2 ** 40
 PiB = 2 ** 50
+
+
+def _os_only(tc, os_name):
+    """
+    Enable test case (TEST[number] class) only on specific OS.
+    Internal helper function.
+    """
+    if sys.platform.startswith(os_name):
+        tc.enabled = True
```

Comment:
This will reenable test disabled by previous check/filter/whatever. I think we should set "enabled" to "true" only in at initialization time.

Reply:
Done.